packages/client/src/components/ui/Snackbar/Snackbar.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from 'react'
+import {
+  StyledSnackbar,
+  StyledText,
+  StyledCloseIcon,
+  StyledTypeIcon,
+} from './StyledComponents'
+import { useAppDispatch, useAppSelector } from '@/hooks'
+import { setSnackbar } from '@/store/reducers/snackbarSlice'
+import { closeicon, okicon, warningicon, attentionicon } from '../../../image'
+export const Snackbar = () => {
+  const dispatch = useAppDispatch()
+
+  const { isOpen, message, type } = useAppSelector(
+    state => state.snackbarReducer
+  )
+
+  let timerToClose: any
```

Comment:
Это число. Думаю можно задать 0 как начальное значение, чтобы убрать any и не было undefined
```suggestion
  let timerToClose = 0
```